lab/lab2/js/part1-ajax-calls.js

```diff
@@ -9,3 +9,16 @@ var Stamen_TonerLite = L.tileLayer('http://stamen-tiles-{s}.a.ssl.fastly.net/ton
   maxZoom: 20,
   ext: 'png'
 }).addTo(map);
+
+
+request = $.ajax ('https://raw.githubusercontent.com/CPLN692-MUSA611-Open-Source-GIS/datasets/master/json/philadelphia-crime-snippet.json')
+request.done (function(res) {
+  console.log(res)
+  crime=JSON.parse(res)
+})
+
+function add_marker (item) {
+  L.marker([item.Lat, item.Lng]).addTo(map)
```

Comment:
Again everything else works but the markers weren't showing up on my map...couldn't figure out why